packages/app/src/systems/Transaction/utils/date.ts

```diff
@@ -0,0 +1,60 @@
+type DateDifference = {
```

Comment:
Can we use some date lib to do this things?

Reply:
alright, pulling in `day-js`